spec/models/institution_builder_spec.rb

```diff
@@ -891,6 +891,15 @@
       end
     end
 
+    describe 'when generating institution programs' do
+      let(:institution_program) { InstitutionProgram.first }
+
+      it 'properly generates institution programs from weams data' do
+        create :program
+        expect { InstitutionBuilder.run(user) }.to change { InstitutionProgram.count }.from(0).to(1)
```

Comment:
Can we get one `expect` verifying the `version` is properly created?
I think it'd be:
`expect(institution_program.version).to eq(Version.current_preview.number)`